qiskit/pulse/configuration.py

```diff
@@ -55,11 +55,9 @@ def __init__(self, name: str | None = None, **params):
         self.params = params
 
     def __repr__(self):
-        return "{}({}{})".format(
-            self.__class__.__name__,
-            "'" + self.name + "', " or "",
-            ", ".join(f"{str(k)}={str(v)}" for k, v in self.params.items()),
-        )
+        name_repr = "'" + self.name + "', " or ""
```

Comment:
I know this was here before -- but isn't that `or` redundant because the first part of  the clause is never `None` or `False`?

Reply:
yeah, makes sense. i'll go ahead and remove!